test/Atc.Rest.ApiGenerator.Tests/Helpers/CodeGenerator/YamlSpecFile.cs

```diff
@@ -0,0 +1,31 @@
+using System.IO;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+
+namespace Atc.Rest.ApiGenerator.Tests.Helpers.CodeGenerator
+{
+    public class YamlSpecFile : IXunitSerializable
+    {
+        public FileInfo FileInfo { get; private set; }
+
+        public string FilePath => FileInfo.FullName;
+
+        public string FileName => FileInfo.Name;
+
+        public string DirectoryName => FileInfo.DirectoryName;
```

Comment:
```suggestion
        public string DirectoryName => FileInfo.DirectoryName ?? throw new InvalidOperationException();
```
or make another kind of fallback or just suppress possible null issue with `!`